src/port1.0/portconfigure.tcl

```diff
@@ -860,6 +860,7 @@ proc portconfigure::get_min_command_line {compiler} {
                     # macOS has supported thread-local storage since Mac OS X Lion.
                     # So __thread (GNU extension) and _Thread_local (C11) could be used.
                     # However, the C++11 keyword was not supported until Xcode 8
+                    #    (https://developer.apple.com/library/archive/releasenotes/DeveloperTools/RN-Xcode/Chapters/Introduction.html#//apple_ref/doc/uid/TP40001051-CH1-SW127)
                     #    (https://developer.apple.com/videos/play/wwdc2016-405/?time=354).
```

Comment:
Should the link to the (exact section of the) release notes replace rather than supplement the link to the WWDC session video?

Reply:
I'd just keep both, too much documentation usually does not hurt.